src/pages/settings/components/inquiry/InquiryDetailPage.tsx

```
@@ -24,37 +33,65 @@ export default function InquiryDetailPage() {
           </div>
-
           <p className="heading-2 mt-20 w-full truncate text-black">
             {data.title}
           </p>
-
+          {data.inquiryImageUrls.length > 0 && (
+            <div className="mt-16 grid w-full grid-cols-3 gap-20">
+              {data.inquiryImageUrls.map((url, index) => (
+                <div
+                  key={`${url}-${index}`}
+                  className="h-139 w-139 overflow-hidden rounded-lg bg-gray-100"
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)